rasa/cli/x.py

```diff
@@ -111,11 +111,28 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
-    endpoints.model = EndpointConfig(
-        "{}/projects/default/models/tags/production".format(rasa_x_url),
-        token=rasa_x_token,
-        wait_time_between_pulls=2,
-    )
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs["wait_time_between_pulls"]
+    custom_url = endpoints.model.url
+
+    if custom_url is not None:
```

Comment:
I would also check this
```suggestion
    if custom_url is not None and custom_url != model_pull_url:
```

Reply:
Ok this should be done, I had that originally but wasn't sure if we actually cared about the custom_url being there or not for this logic.